src/tests/fixtures/comments.fixture.js

```diff
@@ -12,3 +20,25 @@ export const badRequest = {
 export const noTripFound = {
   comment: faker.lorem.sentence()
 };
+
+export const commentToDelete = {
+  id: faker.random.number({ min: 10, max: 15 }),
+  userId: loggedInUser.id,
+  subjectId: newTripComment.id,
+  subjectType: 'Trip',
+  comment: faker.lorem.sentence()
+};
+
+export const commentOfOtherUser = {
```

Comment:
This object is similar to `commentToDelete`. Use spread operator here instead and override the property you want to change.